src/pages/StudentDetailPage/StudentDetailPage.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Grid, Typography, Button } from '@material-ui/core';
+import { Link } from 'react-router-dom';
+import StudentDetailContainer from '../../containers/StudentDetailContainer/StudentDetailContainer';
+const useStyles = makeStyles((theme) => ({
```

Comment:
Добавить пробел между импортом и const
```suggestion

const useStyles = makeStyles((theme) => ({
```

Reply:
@xx3u  Добавлен